hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/dao/NodeInfo.java

```diff
@@ -128,6 +129,7 @@ public NodeInfo(RMNode ni, ResourceScheduler sched) {
       this.usedMemoryOpportGB = opportStatus.getOpportMemoryUsed();
       this.usedVirtualCoresOpport = opportStatus.getOpportCoresUsed();
       this.numQueuedContainers = opportStatus.getQueuedOpportContainers();
+      this.numPausedContainers = opportStatus.getPausedOpportContainers();
```

Comment:
Please take look at all the test failures

Reply:
@bibinchundatt Thanks for the review. UT seems to be passing now. Please take a look.